packages/resource-deployment/scripts/batch-pool-enable-msi.sh

```diff
@@ -77,6 +77,32 @@ getVmssInfo() {
 assignSystemIdentity() {
     for vmssResourceGroup in "${vmssResourceGroups[@]}"; do
         echo "Enabling system-assigned managed identity for VMSS resource group $vmssResourceGroup"
+
+        # Wait until we are certain the resource group exists
+        waiting=false
+        timeout=1800 # timeout after half an hour
```

Comment:
Is this a case to wait for 30 mins?

Reply:
Yes, last week it was taking between 15 minutes and several hours for the resource group to become visible. There's a support ticket out right now for why it's taking so long, but we can reduce the timeout once that gets sorted out.

Reply:
Please keep related bug open to have the time reduced when Azure will have resolution.